### ISSUE TITLE
Accessibility Feedback

### ISSUE THREAD

**Issue by Buwujiu** [Pan-Canadian-Genome-Library/daco#667]
> Really wonderful work team, the website is looking great! Most critical issues from AXE have been resolved along with keyboard functionality. :)
> 
> Just a couple of minor updates:
> 1. The red colour we are using for error messaging does not have enough contrast for AA accessibility, please use: #E7131A instead
> 2. I can't use the TAB button on the keyboard to access the footer. 
> 
> Thanks team! 
> 
> @Buwujiu @demariadaniel 
>  

 _Originally posted by @patrickdsantos in [#431](https://github.com/Pan-Canadian-Genome-Library/daco/issues/431#issuecomment-4509323688)_
